index.html

```diff
@@ -163,67 +189,114 @@ <h2>Profesional Art <br>Printing Data</h2>
                <li><button class="projects-btn" type="button">Ruby</button></li>
            </ul>
        </div>
-    </div>  
-                         </div>
-                        <button class="see-project">See project</button>
-                    </div>
-                </div>
-            </div>
+    </div> 
```

Comment:
to make your code more readable, please try to use indentation for all your tags, the parent tags, and their children, if you are using vs code as a text editor, a feature called document format will prove quite helpful, other than that great job on your tremendous effort